microsoft-edge/webview2/roadmap.md

```diff
@@ -52,4 +52,4 @@ The .NET SDK has reached GA.
 
 ### Windows UI Library 3
```

Comment:
Not sure how to leave a comment on the rest of this file. But other updates we can do to clean this up:
Line 21 - Remove "UWP Preview" from the future updates section.
Add to end - Add a similar section for "Windows UI Library 2" and mention UWP support there has also reached GA.

If you prefer we can do those in future updates to this page to keep this change lightweight.